unit_tests/unit/events/test_loaders.py

```
@@ -0,0 +1,590 @@
+    event = YcsbStressEvent.error(node=[], stress_cmd="c-s", log_file_name="1.log")
+    assert event.severity == Severity.ERROR
+    assert event.node == "[]"
+    assert event.stress_cmd == "c-s"
+    assert event.log_file_name == "1.log"
+    assert not event.errors
+    event.event_id = "68067fe2-4c9e-421c-97b5-12db8d7ba71d"
+    assert str(event) == (
+        "(YcsbStressEvent Severity.ERROR) period_type=not-set "
+        "event_id=68067fe2-4c9e-421c-97b5-12db8d7ba71d: type=error node=[]\nstress_cmd=c-s"
+    )
+    assert event == pickle.loads(pickle.dumps(event))
```

Ruff (0.15.21)

[error] `pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)